ci-operator/config/jboss-fuse/camel-k-test-container/jboss-fuse-camel-k-test-container-main__camel-k-ocp4.14-lp-rosa.yaml

```diff
@@ -0,0 +1,58 @@
+base_images:
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- context_dir: .
+  dockerfile_path: Dockerfile
+  to: camel-k-runner
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: camel-k-interop-aws-rosa-sts
+  cron: 0 6 25 10 *
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      FIREWATCH_CONFIG: |
+        [
+          {"step": "aws-*", "failure_type": "all", "classification": "Infrastructure AWS", "jira_project": "LPTOCPCI", "group": {"name": "cluster", "priority": 1}},
```

Comment:
For each line of the firewatch config we want to add additional labels for issue tracking purposes

"jira_additional_labels": ["4.14-lp","rosa-lp","product_short_name-lp"]

So for this line it would be
{"step": "aws-*", "failure_type": "all", "classification": "Infrastructure AWS", "jira_project": "LPTOCPCI", "group": {"name": "cluster", "priority": 1}, "jira_additional_labels": ["4.14-lp","rosa-lp","camel-k-lp"]},